src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -85,10 +100,12 @@ TODO: Show metrics specific configuration (i.e MetricReaderOptions).
 * `ExportProcessorType`: Whether the exporter should use [Batch or Simple
   exporting
   processor](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#built-in-span-processors).
-  The default is Batch.
+  The default is Batch. **Note**: This option is only applicable in case of
```

Comment:
You could probably move these two properties to a sub-heading such as "Traces specific properties"

Reply:
This is just temporary. We can remove this completely if we do https://github.com/open-telemetry/opentelemetry-dotnet/issues/4527

Reply:
updated